open-reac/src/main/java/com/powsybl/openreac/parameters/OpenReacAmplIOFiles.java

```diff
@@ -0,0 +1,87 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.openreac.parameters;
+
+import com.powsybl.ampl.executor.AmplInputFile;
+import com.powsybl.ampl.executor.AmplOutputFile;
+import com.powsybl.ampl.executor.AmplParameters;
+import com.powsybl.iidm.network.Network;
+import com.powsybl.openreac.parameters.input.*;
+import com.powsybl.openreac.parameters.input.algo.AlgorithmInput;
+import com.powsybl.openreac.parameters.output.OpenReacResult;
+import com.powsybl.openreac.parameters.output.ReactiveSlackOutput;
+
+import java.util.Collection;
+import java.util.List;
+
+/**
+ * OpenReacAmplIOFiles will interface all inputs and outputs needed for OpenReac to the abtracted Ampl Executor.
+ * <p>
+ * The user of OpenReac should not see this class directly. One should use {@link OpenReacParameters} for inputs
+ * and {@link OpenReacResult} for outputs.
+ * However, when adding new inputs (outputs) to OpenReac, one must add {@link AmplOutputFile} (@link AmplInputFile)
+ * here through {@link OpenReacAmplIOFiles#getInputParameters} ({@link OpenReacAmplIOFiles#getOutputParameters})
+ *
+ * @author Nicolas Pierre <nicolas.pierre at artelys.com>
+ */
+public class OpenReacAmplIOFiles implements AmplParameters {
```

Comment:
Maybe OpenReacAmplParameters would be more general

Reply:
We already have `OpenReacParameters` and having it with `OpenReacAmplParameters` might be confusing.

This class is the interface between business parameters and files that will be read by ampl. I don't have a good idea for this interface